code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -4,8 +4,15 @@
 		stat = CONSCIOUS
 		return
 	health = maxHealth - (getBruteLoss() + getFireLoss())

```

Comment:
Wait, I assume you're talking about life.dm having another updateHealth proc, yes?